docs/developer-docs/latest/setup-deployment-guides/configurations/databases/mongodb.md

```diff
@@ -288,10 +282,16 @@ module.exports = ({ env }) => ({
     default: {
       connector: 'mongoose',
       settings: {
-        uri: env('DATABASE_URI'),
+        host: env('DATABASE_HOST'),
```

Comment:
We should add tabs here to show the two ways to configure the database file for either a URI or a breakdown as both work